prisma/schema.prisma

```diff
@@ -0,0 +1,32 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:./dev.db"
+}
+
+model Measurement {
+  date                       DateTime @unique
+  componentAFlowKgH          Float
+  componentBFlowKgH          Float
+  coolantTemperatureC        Float
+  reactorOutletTemperatureC  Float
+  reactorHotspotTemperatureC Float
+  yieldPercent               Float
+  productFlowTonsPerDay      Float
+  notes                      Note[]
+}
```

Comment:
Sure. If it were likely that:

-  the number of metrics would increase over time
- assuming that each metric is of the same type

then in my view would be convenient to use generic schema for any metric.

Example schema:
```typescript
// index (timestamp, metricId)
type Measurement = { timestamp: Date, metricId: number, value: number };

type Metric = { metricId: number, name: string };

type Note = { id: number, timestamp: Date, type: NoteType, body: string };

// number is more space-effective than string
enum NoteType { Alert, Comment };

// many-to-many relation between Note and Measurement
type NoteMeasurement = {noteId: number, metricId: number, timestamp: Date }; // *
```

It would:
- avoid database migrations (depends on database)
- allow to add new metrics dynamically without code changes
- probably make code cleaner
- unlock possibility to track metrics independently on different time intervals
- allow to ingest metrics data in parallel without synchronization (for example from many topics/shards of Kafka)